src/stress/adapter/device_allocation.spec.ts

```diff
@@ -0,0 +1,264 @@
+export const description = `
+Stress tests for GPUAdapter.requestDevice.
+`;
+
+import { Fixture } from '../../common/framework/fixture.js';
+import { makeTestGroup } from '../../common/framework/test_group.js';
+import { attemptGarbageCollection } from '../../common/util/collect_garbage.js';
+import { getGPU } from '../../common/util/navigator_gpu.js';
+import { assert, iterRange } from '../../common/util/util.js';
+import { kAdapterTypes, kAdapterTypeOptions } from '../../webgpu/capability_info.js';
+import { DefaultLimits } from '../../webgpu/constants.js';
+
+class DeviceAllocationTests extends Fixture {
+  async createDeviceAndComputeCommands(adapter: GPUAdapter) {
```

Comment:
nit: could be helpful to comment that we're only trying to make a valid pipeline, but we're never going to actually run it.

except.. maybe it's also worth having a .destroy test that does queue.submit([]) of everything, immediately followed by destroy? i.e. destroy while GPU executing

Reply:
Added comments for each of the helpers.